Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -176,75 +176,80 @@ extension JNISwift2JavaGenerator {
   }
 
   private func printFunctionBinding(_ printer: inout CodePrinter, _ decl: ImportedFunc) {
-    guard let _ = translatedDecl(for: decl) else {
+    guard let translatedDecl = translatedDecl(for: decl) else {
       // Failed to translate. Skip.
       return
     }
 
+    var modifiers = "public"
```

Comment:
Might be better as a list

```suggestion
    var modifiers = ["public"]
```

so we don't have to do the weird `" "` in the append below